app/models/poll_email_info.rb

```diff
@@ -84,7 +84,7 @@ def utm_hash(args = {})
   private
 
   def membership
-    @membership ||= poll.guest_group.memberships.find_by(user: recipient)
+    @membership ||= poll.guest_group.memberships.pending.find_by(user: recipient)
```

Comment:
To my eye the only place we use poll_email_info#membership is in `target_url`; if they click on a link to a used membership it'll fly through `InvitationAlreadyUsed` and land on `membership.target_model` anyway, so I'm not sure if this introduces a change in behaviour.